src/desktops/dskSelectMap.cpp

```diff
@@ -91,6 +96,10 @@ dskSelectMap::dskSelectMap(const CreateServerInfo& csi)
     AddTextButton(4, 590, 530, 200, 22, TC_GREEN2, _("Load game..."), NormalFont);
     // "Weiter"
     AddTextButton(5, 590, 560, 200, 22, TC_GREEN2, _("Continue"), NormalFont);
+    // random map generation
+    AddTextButton(6, 380, 530, 150, 22, TC_GREEN2, _("Create Random"), NormalFont);
+    // random map settings
+    AddTextButton(7, 540, 530, 40, 22, TC_GREEN2, _("..."), NormalFont);
```

Comment:
But you can set basic addon settings in the options menu and still change it when hosting. 
Just think of the following: you found some players and instead of just hosting and changing some settings for the random map you first have to leave the host menu, change options, rehost, damn forgot something... Etc. 
But thats just something I mind